service/application.go

```diff
@@ -324,3 +337,20 @@ func (app *Application) createMemoryBallast() ([]byte, uint64) {
 	}
 	return nil, 0
 }
+
+// updateActiveConfig shutdown the currently active configuration and loads and starts a new one.
```

Comment:
```suggestion
// updateActiveConfig shutdowns the currently active configuration and loads and starts a new one.
```

Reply:
I don't quite understand the comment. Does this shutdown the configuration? Or was this supposed to say "currently active service"? I am a bit confused.

Reply:
Good point. I think that now it makes more sense to call it `updateService`, since it reflects better to the app internals. Let me know what you think.